guardian/braze-components
# Support Node 20.9 LTS as an engine

## What does this change?

Add support for Node 20 (Iron) as an engine. This is also an LTS version of Node [as of today](https://nodejs.org/en/blog/release/v20.9.0).

## How to test

Everything should run as-is, but [you can confirm using this online checker](https://jubianchi.github.io/semver-check/#/^18.15%20||%20^20.9/20.9).

## How can we measure success?

Consumer of this package should not be constrained to stay on Node 18, which is now in maintenance mode.

## Have we considered potential risks?

It would probably make more sense to use an unbound upper range, [such as `>=18.15`](https://jubianchi.github.io/semver-check/#/%3E%3D18.15/20.9).

## Images

N/A
